src/ModelSelector.jsx

```diff
@@ -0,0 +1,27 @@
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormHelperText from '@mui/material/FormHelperText';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
+
+export default function ModelSelector({ handleSelector, model }) {
+    return (
+        <FormControl sx={{ m: 1, minWidth: 60 }}>
+            <InputLabel id="model">Model</InputLabel>
```

Comment:
I didn't check if these ids actually get produced in the DOM, but it's probably best to avoid reusing the same id for different elements.
```suggestion
            <InputLabel id="modelLabel">Model</InputLabel>
```